cmd/describe/cluster/cmd.go

```diff
@@ -120,6 +120,11 @@ func run(cmd *cobra.Command, argv []string) {
 			}
 			phase = "(" + errorCode + "Install is taking longer than expected)"
 		}
+	case cmv1.ClusterStateError:
+		status, _ := r.OCMClient.GetClusterStatus(cluster.ID())
```

Comment:
You don't need to get the cluster status, the description should already be present in cluster.Status() (but perhaps we need to update the sdk if it isn't there). That's probably also true for other places where we currently use `GetClusterStatus`.

Reply:
Please take a second look, tested with error state and it did work

Reply:
Just tested with waiting state and seems fine, waiting for install to complete

Reply:
Seems fine, let me know what you think later